app/data/battlepets.json

```diff
@@ -702,6 +702,48 @@
         ],
         "name": "Operation: Mechagon"
       },
+      {
+        "id": "dbddd3a4",
+        "items": [
+          {
+            "creatureId": 162499,
+            "icon": "inv_icon_wing03d",
+            "name": "Oasis Void-Duster",
+            "spellid": null
+          },
+          {
+            "creatureId": 154796,
+            "icon": "inv_pet_cockroach",
+            "name": "Void-Scarred Locust",
+            "spellid": null
+          }
+        ],
+        "name": "Assault: Uldum"
+      },
+      {
+        "id": "0d5fc9cd",
+        "items": [
+          {
+            "creatureId": 162525,
+            "icon": "inv_pet_cricket",
+            "name": "Golden Chirper",
+            "spellid": null
+          },
+          {
+            "creatureId": 162502,
```

Comment:
And this is the ID of the Void-Scarred Locust, not the Voidglower.

Reply:
Fixed.